src/common/ecs-features/constants.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
Incorrect information. There is no license.txt in the root

Reply:
Correct - seems this needs to be fixed in all the files in our extension. Will follow-up and raise a seperate PR for that.

Reply:
Please mark conversation as resolved only after actually making the change

Reply:
Resolving as this should be taken as a corretive change across repo and is out of scope for this PR.